### DISCUSSION TITLE
Codemagic authentication key cames in 20m so its already expired.

### DISCUSSION THREAD

**Discussion by pashkash** [codemagic-ci-cd/codemagic-docs#2286] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

1. Trying to login by entering email.
2. Waiting for the code 20min (always 20m)
3. Use the code
4. Getting message that the code is expired

### Expected results

I expect to login with the code.

### Actual results

Getting message that the code is expired

**Comment by mikhail-tokarev** [comment]
hey
I can see that your SMTP server (`@shv*****.tech`) responses with 451 code to all Codemagic emails and it causes 20min delay. You need to add `codemagic.io` domain to the white list or configure another limits on your side.

As an alternative solution you can sign in via GitHub, but GitHub account should be linked to the same email address.

**Comment by pashkash** [comment]
ty man; entered via github; happy new year and many thanks
